### PR TITLE
Bump scala / scala-js / scala-native versions

### PR DESCRIPTION
**Pull request by alexarchambault**

### CODE REVIEW

**Comment by lihaoyi** [comment]
Gave you commit access to this repo too; feel free to make use of it merge your PRs, whether your own or other people's

**Comment by alexarchambault** [comment]
Thanks @lihaoyi. I was planning on pinging @olafurpg for that ^^

File: project/build.sbt [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-    "org.scala-native" % "sbt-scalajs-crossproject" % "0.2.0"
-  else
-    "org.scala-native" % "sbt-crossproject" % "0.2.0"
-})
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.1")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "0.6.0")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.3.8")
+addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "0.6.0")
+
+addSbtCoursier
```
Comment (olafurpg) [inline project/build.sbt]: woah, I didn't know about this 😍

**Comment by alexarchambault** [comment]
@olafurpg @lihaoyi How do you guys feel about temporarily removing scala-js 1.x support, and re-instating in a subsequent PR? I keep running into weird errors with both `1.0.0-M4` and `1.0.0-M5`, that I can explain in more detail in the PR re-enabling it (unexpectedly missing scala-js artifacts, linking errors, …).

File: project/build.sbt (hunk 1 above)
Comment (alexarchambault) [reply project/build.sbt]: I should mention it in the coursier README…

**Comment by olafurpg** [comment]
Fine by me 👍

**Pull request merged**